services/QuillLMS/db/migrate/20221109181742_create_teacher_info.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateTeacherInfo < ActiveRecord::Migration[6.1]
+  def change
+    create_table :teacher_infos do |t|
+      t.integer :minimum_grade_level
+      t.integer :maximum_grade_level
+      t.references :teacher, foreign_key: { to_table: :users }
```

Comment:
Can teacher ever null for a valid saved object?

If not, consider validates presence for them and making the DB fields non-null.

(basically a [comment](https://github.com/empirical-org/Empirical-Core/pull/9866/files#r1001850632) Dan wrote on one of my recent PRs)